siad/main.go

```diff
@@ -132,6 +132,9 @@ Explorer (e):
 
 // main establishes a set of commands and flags using the cobra package.
 func main() {
+	if build.DEBUG {
+		fmt.Println("Running with debugging enabled")
+	}
```

Comment:
Let's do the easy thing for now and we can extend the API in another PR. But having 'version', 'release', and also 'race' would be nice. As well as 'Debug'.